collections-arrays/HowToEfficientlyRandomizeAnArray/HowToEfficientlyRandomizeAnArray/ArrayFunctions.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace HowToEfficientlyRandomizeAnArray
+{
+    public class ArrayFunctions
+    {
+        private readonly Random _rnd;
+
+        public ArrayFunctions()
```

Comment:
No need for the constructor, you can initialize _rnd directly at the declaration

Reply:
Hi Jeff, I'm not really familiar with `Random.Shared` but very good pointer! What would you say would be the best approach:
1. Declare `private static Random _rnd = Random.Shared` and then use it
2. Just call `Random.Shared` directly where needed - `array.OrderByDescending(x => Random.Shared.Next()).ToArray();`

Reply:
I would just call it directly, that way it is clear and you can simplify the code by removing the member variable. It doesn't really increase the character count too much in the other locations. This is a good article talking about Random and the Random.Shared introduced in .NET 6.0: https://andrewlock.net/building-a-thread-safe-random-implementation-for-dotnet-framework/

Reply:
Thanks, this is very helpful!